gcb/makestage/cloudbuild.yaml

```diff
@@ -21,7 +21,7 @@ steps:
   - |
     set -eu -o pipefail
     make vendor-go
-    make CMREL_KEY="${_KMS_KEY}" RELEASE_TARGET_BUCKET="${_RELEASE_TARGET_BUCKET}" -j16 upload-release
+    make CMREL_KEY="${_KMS_KEY}" RELEASE_TARGET_BUCKET="${_RELEASE_TARGET_BUCKET}" -j1 upload-release
```

Comment:
suggestion: If we're gonna reduce parallelism as a solution (rather than try to fix the issue for next time) should we reduce the size of the machine we're using (`options.machineType` at the bottom) to a smaller type with fewer cores? Feels excessive having 32 cores if we only use one!

Reply:
The 32 cores will must help speed up the go builds so I'd prefer to make the minium viable change here and revert it once we've fixed the underlying issue.

Reply:
That's reasonable, yeah!